src/language/lexer.d.ts

```diff
@@ -1,4 +1,4 @@
-import { syntaxError } from '../error';
```

Comment:
Please extract all changes in `.d.ts` into PR against `master`.
Please keep `d.ts` files in sync with flow files:
https://github.com/graphql/graphql-js/blob/85d4c9940f7113789b3bd1b62606f382ace5472b/src/language/lexer.js#L5

Also, we never use `index` files inside the library only direct imports from files.